graphrag_sdk/tests/test_facade.py

```
@@ -270,6 +270,36 @@ async def test_deduplicate_entities_no_duplicates(self, mock_conn, embedder, llm
+
+        g = GraphRAG(connection=mock_conn, llm=llm, embedder=embedder, schema=sample_schema)
+        extractor = g._default_extractor()
+        assert isinstance(extractor, SchemaGuidedExtraction)
+
+    async def test_ingest_explicit_extractor_overrides(self, mock_conn, embedder, llm, sample_schema):
+        """Explicit extractor param should override _default_extractor."""
+        from graphrag_sdk.ingestion.extraction_strategies.merged_extraction import MergedExtraction
+
+        g = GraphRAG(connection=mock_conn, llm=llm, embedder=embedder, schema=sample_schema)
+        # Even with a populated schema, explicit MergedExtraction should be used
+        custom = MergedExtraction(llm=llm, embedder=embedder)
```

Ruff (0.15.4)

[error] Local variable `custom` is assigned to but never used

Remove assignment to unused variable `custom`

(F841)